src/pages/VolunteersCampaignWizard.page

```diff
@@ -15,107 +15,144 @@
+                    <div class="slds-spinner__dot-a"></div>
+                    <div class="slds-spinner__dot-b"></div>
+                </div>
+            </div>
+            <apex:actionStatus id="statusLoad"
+                onstart="document.getElementById('divLoading').style.display = 'block';"
+                onstop="document.getElementById('divLoading').style.display = 'none';"/>
 
-                <apex:pageBlockSectionItem rendered="{!campaignIdClone != null}" >
-                    <apex:outputText value="{!$Label.labelVolunteersWizardOverrideHoursStatus}" />
-                    <apex:inputField value="{!hoursStatus.Status__c}" />                	
-				</apex:pageBlockSectionItem>
+            <div class="slds-page-header" role="banner">
+              <h1 class="slds-text-heading--large">{!$Label.labelVolunteersWizardTitle}</h1>
+              <p class="slds-text-body--small" style="font-style:italic;">{!$Label.labelVolunteersWizardHelp}</p>
+            </div>
+            <apex:outputPanel id="panelErrors">
+                <c:UTIL_PageMessages html-class="slds-grid slds-grid--align-center slds-m-bottom--small"/>
+            </apex:outputPanel>
 
-            </apex:pageBlockSection>
-        </apex:pageBlock>
-    </apex:form>    
+            <article class="slds-card">
+                <div class="slds-card__header slds-grid">
+                    <div class="slds-col slds-text-heading--medium">
+                        {!$Label.labelVolunteersWizardNewCampaignTitle}
+                    </div>
+                    <div class="slds-col slds-align-middle">
+                        <div class="slds-button-group" role="group">
+                            <apex:commandButton styleClass="slds-button slds-button--brand" value="{!$Label.labelButtonSave}" action="{!Save}"/>
+                            <apex:commandButton styleClass="slds-button slds-button--neutral" value="{!$Label.labelButtonCancel}" immediate="true" action="{!URLFOR($Action.Volunteer_Job__c.Tab,$ObjectType.Volunteer_Job__c)}"/>
+                        </div>
+                    </div>
+                </div>
+                <div class="slds-card__body">
+                    <div class="slds-p-left--medium slds-p-bottom--medium">
+                        <div class="slds-text-title--caps">{!$Label.labelVolunteersWizardCampaignInformation}</div>
+                    </div>
+                    <div class="slds-p-left--medium slds-p-right--medium slds-theme--default">
+                        <form class="slds-form--horizontal">
+                            <apex:repeat value="{!$ObjectType.Campaign.FieldSets.VolunteersWizardFS}" var="f">
+                                <div class="slds-form-element">
+                                    <apex:outputLabel value="{!f.Label}" for="idInputField" styleClass="slds-form-element__label"/>
+                                    <div class="slds-form-element__control">
```

Comment:
That did the trick. The error displays twice, but I know we've been unable to work around that in the past.